spec/system/provider_interface/provider_changes_a_course_before_point_of_offer_spec.rb

```diff
@@ -35,35 +35,31 @@
     and_i_click_continue
     then_i_see_a_list_of_courses_to_select_from
 
-    when_i_select_a_course_with_one_study_mode
+    when_i_select_a_different_course
+    and_i_click_continue
+    then_no_study_mode_is_pre_selected
+
+    when_i_select_a_study_mode
     and_i_click_continue
     then_i_am_taken_to_the_change_location_page
-  end
 
-  scenario 'Changing a course choice before point of offer' do
-    given_i_am_a_provider_user
-    and_the_feature_flag_is_enabled
-    and_i_am_permitted_to_make_decisions_for_my_provider
-    and_i_sign_in_to_the_provider_interface
-    and_the_provider_has_multiple_courses
-    and_the_provider_user_can_offer_multiple_provider_courses
+    when_i_select_a_new_location
+    and_i_click_continue
+    then_the_review_page_is_loaded
 
-    when_i_visit_the_provider_interface
-    and_i_click_an_application_choice_that_is_interviewing
-    and_i_click_on_change_the_training_provider
-    then_i_see_a_list_of_training_providers_to_select_from
+    when_i_click_change_course
+    then_i_am_taken_to_the_change_course_page
 
-    when_i_select_a_different_provider
+    when_i_select_a_course_with_one_study_mode
     and_i_click_continue
-    then_i_see_a_list_of_courses_to_select_from
-
-    when_i_select_a_different_course
+    and_i_select_a_new_location
     and_i_click_continue
-    then_no_study_mode_is_pre_selected
+    then_the_review_page_is_loaded
 
-    when_i_select_a_study_mode
+    when_i_click_change_course
```

Comment:
This system spec is getting unwieldy. 
I would consider either testing this elsewhere (ideally a request spec with the various change variants) or splitting system spec into the happy path and the change paths.

Reply:
Yeah, this is a good point – i think I'll split this out into two separate paths once the service is in place